node.gypi

```diff
@@ -288,7 +288,16 @@
     [ 'OS=="sunos"', {
       'ldflags': [ '-Wl,-M,/usr/lib/ld/map.noexstk' ],
     }],
-
+    [ 'OS in "freebsd linux"', {
+      'ldflags': [ '-Wl,-z,relro',
+                   '-Wl,-z,now' ]
+    }],
```

Comment:
Is this change intentional? If so why is it not in a separate commit (backporting this https://github.com/nodejs/node/commit/2d4dd108296d327b874492fbd55ebbe00d003ef4)?

Reply:
@lundibundi It was not intentional. I'll fix this and update the PR.

Reply:
@lundibundi Fixed the accidental deletion of blank line.